### ISSUE TITLE
First platform build results in VS.NET error

### ISSUE THREAD

**Issue by Woland2k** [VirtoCommerce/vc-platform#1854]
Latest source code (RC6?) compiles but displays the following error in VS.NET on the first run:

**Steps to reproduce**
1. Create an empty vc-platform directory
2. Run the following command: git clone -b release/3.0.0 https://github.com/VirtoCommerce/vc-platform.git vc-platform
3. Open in VS.NET and click compile

```
=====================
4/28/2020 10:18:03 AM
Recoverable
System.AggregateException: Project system data flow 'CrossTarget Joint Input: 42250221' closed because of an exception: System.AggregateException: One or more errors occurred. ---> System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.ThrowIfChanged()
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.MoveNext()
   at System.Collections.Immutable.ImmutableDictionary`2.Builder.<get_Keys>d__18.MoveNext()
   at System.Linq.Enumerable.<ExceptIterator>d__73`1.MoveNext()
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.<FromChanges>g__SyncTargetFrameworks|1_0(<>c__DisplayClass1_0& )
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.FromChanges(String projectPath, DependenciesSnapshot previousSnapshot, ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, ImmutableArray`1 snapshotFilters, IReadOnlyDictionary`2 subTreeProviderByProviderType, IImmutableSet`1 projectItemSpecs)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<>c__DisplayClass34_0.<UpdateDependenciesSnapshot>b__0(DependenciesSnapshot previousSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.SnapshotUpdater.TryUpdate(Func`2 updateFunc, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.UpdateDependenciesSnapshot(ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<get_Subscribers>g__OnSubscriberDependenciesChanged|29_0(Object sender, DependencySubscriptionChangedEventArgs e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.RaiseDependenciesChanged(ITargetFramework targetFramework, IDependenciesChanges changes, AggregateCrossTargetProjectContext currentAggregateContext, IProjectCatalogSnapshot catalogSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriber.Handle(AggregateCrossTargetProjectContext currentAggregateContext, ITargetFramework targetFrameworkToUpdate, Tuple`3 e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.<>c__DisplayClass12_0.<<OnProjectChangedAsync>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.<>c__DisplayClass13_0.<<LoadedProjectAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<>c__DisplayClass3_0.<<ExecuteAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.AwaitExtensions.ExecuteContinuationSynchronouslyAwaiter.GetResult()
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<ExecuteAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.OnceInitializedOnceDisposedUnderLockAsync.<ExecuteUnderLockCoreAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.DataReceivingBlockSlim`1.<ProcessInputQueueAsync>d__5.MoveNext()
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.ThrowIfChanged()
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.MoveNext()
   at System.Collections.Immutable.ImmutableDictionary`2.Builder.<get_Keys>d__18.MoveNext()
   at System.Linq.Enumerable.<ExceptIterator>d__73`1.MoveNext()
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.<FromChanges>g__SyncTargetFrameworks|1_0(<>c__DisplayClass1_0& )
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.FromChanges(String projectPath, DependenciesSnapshot previousSnapshot, ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, ImmutableArray`1 snapshotFilters, IReadOnlyDictionary`2 subTreeProviderByProviderType, IImmutableSet`1 projectItemSpecs)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<>c__DisplayClass34_0.<UpdateDependenciesSnapshot>b__0(DependenciesSnapshot previousSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.SnapshotUpdater.TryUpdate(Func`2 updateFunc, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.UpdateDependenciesSnapshot(ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<get_Subscribers>g__OnSubscriberDependenciesChanged|29_0(Object sender, DependencySubscriptionChangedEventArgs e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.RaiseDependenciesChanged(ITargetFramework targetFramework, IDependenciesChanges changes, AggregateCrossTargetProjectContext currentAggregateContext, IProjectCatalogSnapshot catalogSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriber.Handle(AggregateCrossTargetProjectContext currentAggregateContext, ITargetFramework targetFrameworkToUpdate, Tuple`3 e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.<>c__DisplayClass12_0.<<OnProjectChangedAsync>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.<>c__DisplayClass13_0.<<LoadedProjectAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<>c__DisplayClass3_0.<<ExecuteAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.AwaitExtensions.ExecuteContinuationSynchronouslyAwaiter.GetResult()
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<ExecuteAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.OnceInitializedOnceDisposedUnderLockAsync.<ExecuteUnderLockCoreAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.DataReceivingBlockSlim`1.<ProcessInputQueueAsync>d__5.MoveNext()<---
. ---> System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.ThrowIfChanged()
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.MoveNext()
   at System.Collections.Immutable.ImmutableDictionary`2.Builder.<get_Keys>d__18.MoveNext()
   at System.Linq.Enumerable.<ExceptIterator>d__73`1.MoveNext()
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.<FromChanges>g__SyncTargetFrameworks|1_0(<>c__DisplayClass1_0& )
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.FromChanges(String projectPath, DependenciesSnapshot previousSnapshot, ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, ImmutableArray`1 snapshotFilters, IReadOnlyDictionary`2 subTreeProviderByProviderType, IImmutableSet`1 projectItemSpecs)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<>c__DisplayClass34_0.<UpdateDependenciesSnapshot>b__0(DependenciesSnapshot previousSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.SnapshotUpdater.TryUpdate(Func`2 updateFunc, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.UpdateDependenciesSnapshot(ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<get_Subscribers>g__OnSubscriberDependenciesChanged|29_0(Object sender, DependencySubscriptionChangedEventArgs e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.RaiseDependenciesChanged(ITargetFramework targetFramework, IDependenciesChanges changes, AggregateCrossTargetProjectContext currentAggregateContext, IProjectCatalogSnapshot catalogSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriber.Handle(AggregateCrossTargetProjectContext currentAggregateContext, ITargetFramework targetFrameworkToUpdate, Tuple`3 e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.<>c__DisplayClass12_0.<<OnProjectChangedAsync>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.<>c__DisplayClass13_0.<<LoadedProjectAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<>c__DisplayClass3_0.<<ExecuteAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.AwaitExtensions.ExecuteContinuationSynchronouslyAwaiter.GetResult()
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<ExecuteAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.OnceInitializedOnceDisposedUnderLockAsync.<ExecuteUnderLockCoreAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.DataReceivingBlockSlim`1.<ProcessInputQueueAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.Rethrow(Exception ex)
   at Microsoft.VisualStudio.ProjectSystem.ProjectErrorReporting.<>c__DisplayClass6_0.<SubmitErrorReport>b__0()
   at Microsoft.VisualStudio.ProjectSystem.ExceptionFilter.<>c__DisplayClass2_0.<Guard>g__action|0()
   at GuardMethodClass.GuardMethod(Func`1 , Func`2 , Func`2 )
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.ThrowIfChanged()
   at System.Collections.Immutable.ImmutableDictionary`2.Enumerator.MoveNext()
   at System.Collections.Immutable.ImmutableDictionary`2.Builder.<get_Keys>d__18.MoveNext()
   at System.Linq.Enumerable.<ExceptIterator>d__73`1.MoveNext()
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.<FromChanges>g__SyncTargetFrameworks|1_0(<>c__DisplayClass1_0& )
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot.FromChanges(String projectPath, DependenciesSnapshot previousSnapshot, ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, ImmutableArray`1 snapshotFilters, IReadOnlyDictionary`2 subTreeProviderByProviderType, IImmutableSet`1 projectItemSpecs)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<>c__DisplayClass34_0.<UpdateDependenciesSnapshot>b__0(DependenciesSnapshot previousSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.SnapshotUpdater.TryUpdate(Func`2 updateFunc, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.UpdateDependenciesSnapshot(ITargetFramework changedTargetFramework, IDependenciesChanges changes, IProjectCatalogSnapshot catalogs, ImmutableArray`1 targetFrameworks, ITargetFramework activeTargetFramework, CancellationToken token)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.<get_Subscribers>g__OnSubscriberDependenciesChanged|29_0(Object sender, DependencySubscriptionChangedEventArgs e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.RaiseDependenciesChanged(ITargetFramework targetFramework, IDependenciesChanges changes, AggregateCrossTargetProjectContext currentAggregateContext, IProjectCatalogSnapshot catalogSnapshot)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriber.Handle(AggregateCrossTargetProjectContext currentAggregateContext, ITargetFramework targetFrameworkToUpdate, Tuple`3 e)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.DependencyRulesSubscriberBase`1.<>c__DisplayClass12_0.<<OnProjectChangedAsync>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.<>c__DisplayClass13_0.<<LoadedProjectAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<>c__DisplayClass3_0.<<ExecuteAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.AwaitExtensions.ExecuteContinuationSynchronouslyAwaiter.GetResult()
   at Microsoft.VisualStudio.Threading.ReentrantSemaphore.StackSemaphore.<ExecuteAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.OnceInitializedOnceDisposedUnderLockAsync.<ExecuteUnderLockCoreAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.DataReceivingBlockSlim`1.<ProcessInputQueueAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.Rethrow(Exception ex)
   at Microsoft.VisualStudio.ProjectSystem.ProjectErrorReporting.<>c__DisplayClass6_0.<SubmitErrorReport>b__0()
   at Microsoft.VisualStudio.ProjectSystem.ExceptionFilter.<>c__DisplayClass2_0.<Guard>g__action|0()
   at GuardMethodClass.GuardMethod(Func`1 , Func`2 , Func`2 )<---

===================

```

**Comment by yecli** [comment]
Problem confirmed.
Seems it was VS inner failure.
Tried described steps on Windows 10 Pro x64, VS 2019 Enterprise 16.5.1. No errors occurred.
Updated to the latest VS 2019 Enterprise 16.5.4 - got this one.

Will check VS bugs regarding this.

**Comment by yecli** [comment]
Here is the VS issue: https://github.com/dotnet/project-system/issues/6090
Should be released in the next 16.5 VS2019 update.
Better to not update VS to 16.5.4 to avoid before the next version will be out.

**Comment by yecli** [comment]
Not fixed in VS 16.5.5.
MS says it will be fixed only in VS 16.6.

**Comment by yecli** [comment]
Fixed in VS 16.6.1
